pages/mint-nft/index.vue

```diff
@@ -899,4 +909,14 @@ function onDownloadNftsCSV (e?: Event) {
   downloadFile({ data: nftMintListData.value, fileName: 'nfts.csv', fileType: 'csv' })
 }
 
+function onSaveISCN (iscnId: string) {
+  if (iscnId) {
+    router.replace({ query: { ...route.query, iscn_id: iscnId } })
+    iscnIdInput.value = iscnId
+    step.value = 1
+  } else {
+    window.location.reload()
```

Comment:
When would `iscnId` be falsy?

Reply:
[✨ Add getter for all records and remove unused function](https://github.com/likecoin/nft-book-press/pull/317/commits/db69b8620e35b4e2b8148d81c251ede51413a290)